analysis/import/TokeiImporter/src/test/kotlin/de/maibornwolff/codecharta/importer/tokeiimporter/TokeiImporterTest.kt

```diff
@@ -19,6 +19,14 @@ class TokeiImporterTest {
         Assertions.assertThat(cliResult).contains(listOf("CHANGELOG.md", "\"loc\":450"))
     }
 
+    @Test
+    fun `reads tokei 12 new json scheme from file`() {
+        val cliResult = executeForOutput("", arrayOf("src/test/resources/tokei_without_inner.json", "--pathSeparator=\\"))
+
+        Assertions.assertThat(cliResult).contains(listOf("CHANGELOG.md", "\"rloc\":450"))
+        Assertions.assertThat(true).isEqualTo(false)
```

Comment:
This test should fail, but I can't run the tests using `./gradlew test`.
I wonder if travis will detect this test.

Reply:
I think I managed to enable the unit tests.
We're using two different testing frameworks in the analysis. JUnit4 or Spek2+JUnit5 (Runner).
Each subproject can override it's test command. I applied both test commands to each subproject.